CONTRIBUTING.md

```diff
@@ -140,6 +140,26 @@ Note: For a list of available gulp commands use:
 
 `$ gulp`
 
+#### Trouble shooting(For Mac)
+
+If you have problem serve the site locally after all these steps, please try the following steps:
+
+Launch Terminal (OS X) and [install RVM](https://rvm.io/rvm/install)
+
+Find a stable version of ruby and install it:
+
+`$ rvm install ruby-VersionNumber`
+
+Use a stable version of ruby for rvm:
+
+`$ rvm use VersionNumber`
```

Comment:
An example for this: rvm use 2.3.1. I will change this command to: "$rvm use [version-number]"